crates/control-interface/src/types/component.rs

```diff
@@ -4,15 +4,15 @@ use std::collections::BTreeMap;
 
 use serde::{Deserialize, Serialize};
 
-use crate::{ComponentId, Result};
+use crate::Result;
 
 /// A summary description of an component within a host inventory
 #[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
 #[non_exhaustive]
 pub struct ComponentDescription {
     /// The unique component identifier for this component
     #[serde(default)]
-    pub(crate) id: ComponentId,
+    pub(crate) id: String,
```

Comment:
I don't think changing this to a string should break anyone, but we should make sure we document it well just in case in the release notes

Reply:
Good point. I'm a little worried about the aliases if they are public, so will do a double check on this one

Reply:
@thomastaylor312 These won't break, but I did keep the string aliases around _just in case_ since they were publicly exported (e.g. https://docs.rs/wasmcloud-control-interface/2.2.0/wasmcloud_control_interface/type.ComponentId.html)